### DISCUSSION TITLE
Change Colors in Blog Post

### DISCUSSION THREAD

**Discussion by AdamFermier** [McShelby/hugo-theme-relearn#556] [Q&A] [answered]
Hi Folks - I have a power user that wants to change some colors of the text - is there a way to do that in markdown - or is that illegal for some reason.  Thanks!  Adam

**Comment by McShelby** [comment, marked as answer]
You can't do this in Markdown and this theme does not contain a shortcode for this. 

> Markdown is not a replacement for HTML, or even close to it

For this, Markdown provides an escape hatch by allowing the users to write raw HTML. In Hugo you have to activate this by adding the following to your config/hugo.toml:

````toml
[markup]
  [markup.goldmark.renderer]
    unsafe = true
````

**Reply by AdamFermier** [reply]
Thank You!
